13-Roman_Data_Models/13-Roman_Data_Models.ipynb

```diff
@@ -0,0 +1,624 @@
+{
```

Comment:
<p>I wold make the steps a list so it's easier to read and understand</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Done, I think...</p>